gui/src/i18n/fr/translation.json

```diff
@@ -383,40 +383,40 @@
       },
       "usb": "Capteur USB",
       "connection-status": {
-        "connecting": "Envoi d'identifiants WiFi",
-        "connected": "Connecté au WiFi",
-        "error": "Impossible de se connecter au WiFi",
+        "connecting": "Envoi d'identifiants Wi-Fi",
+        "connected": "Connecté au Wi-Fi",
+        "error": "Impossible de se connecter au réseau",
         "start-connecting": "Recherche de capteurs",
         "handshake": "Connecté au serveur"
       },
       "connected-trackers": "{{amount}} capteurs connectés",
       "next": "J'ai connecté tous mes capteurs"
     },
     "assign-trackers": {
-      "back": "Revenir aux identifiants Wifi",
+      "back": "Revenir aux identifiants Wi-Fi",
       "title": "Attribuer des capteurs",
-      "description": "Choisissons quel capteur va où.",
-      "assigned": "{{assigned}} de {{trackers}} capteurs attribués",
+      "description": "Choisissons où mettre chaque capteur.",
+      "assigned": "{{assigned}} capteur(s) sur {{trackers}} attribués",
```

Comment:
```suggestion
      "assigned": "{{assigned}} capteur(s) sur {{trackers}} attribué(s)",
```

Reply:
won't {{trackers}} always be >1?

Reply:
no:
![image](https://user-images.githubusercontent.com/20461725/208202516-82dbbb31-d315-464a-8e5d-8fb2c97e314e.png)